plugins/emptycells_plugin.py

```diff
@@ -0,0 +1,20 @@
+""""Standard empty plugin for SheetShuttle."""
```

Comment:
I don't think we should be including the plugin with the code submission. Please make sure to remove this from the PR